_data/locales/ru-RU.yml

```diff
@@ -30,48 +30,61 @@ panel:
 copyright:
   # Shown at the bottom of the post
   license:
-    template: Публикация защищена лицензией :LICENSE_NAME.
+    template: Авторский пост защищен лицензией :LICENSE_NAME.
     name: CC BY 4.0
     link: https://creativecommons.org/licenses/by/4.0/
 
   # Displayed in the footer
   brief: Некоторые права защищены.
   verbose: >-
-    Публикации на сайте защищены лицензией Creative Commons Attribution 4.0 International (CC BY 4.0),
-    если в тексте публикации не указано иное.
+    Если не указано иное, авторские посты на этом сайте защищены лицензией Creative Commons Attribution 4.0 International (CC BY 4.0).
 
-meta: Powered by :PLATFORM with :THEME theme
+meta: Использует тему :THEME для :PLATFORM
 
 not_found:
-  statment: Извините, эта ссылка указывает на ресурс который не существует.
+  statment: Извините, мы перепутали URL-адрес или он указывает на что-то несуществующее.
 
 notification:
   update_found: Доступна новая версия контента.
-  update: Обновлять
+  update: Обновить
 
 # ----- Posts related labels -----
 
 post:
   written_by: Автор
-  posted: Время публикации
+  posted: Опубликовано
   updated: Обновлено
   words: слов
   pageview_measure: просмотров
   read_time:
-    unit: минут
+    unit: мин.
     prompt: чтения
-  relate_posts: Вам также может быть интересно
+  relate_posts: Похожие посты
   share: Поделиться
   button:
-    next: Предыдущая публикация
-    previous: Следующая публикация
+    next: Следующий пост
+    previous: Предыдущий пост
     copy_code:
-      succeed: Скопировано успешно!
+      succeed: Скопировано!
     share_link:
       title: Скопировать ссылку
       succeed: Ссылка успешно скопирована!
 
+# Date time format.
+# See: <http://strftime.net/>, <https://day.js.org/docs/en/display/format>
+df:
+  post:
+    strftime: "%d.%m.%Y"
+    dayjs: "DD.MM.YYYY"
+  archives:
+    strftime: "%B"
+    dayjs: "MMMM"
+
```

Comment:
```suggestion
```

Almost ready to merge, but you should remove the `df.archives`` part because `strftime`` is not as powerful as `dayjs` and doesn't have definitions for months other than English, so the page loads with visual jitters when switching text between the two.